transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollChannelOption.java

```diff
@@ -33,8 +33,7 @@
     public static final ChannelOption<Boolean> IP_TRANSPARENT = valueOf("IP_TRANSPARENT");
     public static final ChannelOption<Boolean> IP_RECVORIGDSTADDR = valueOf("IP_RECVORIGDSTADDR");
     public static final ChannelOption<Integer> TCP_FASTOPEN = valueOf(EpollChannelOption.class, "TCP_FASTOPEN");
-    public static final ChannelOption<Boolean> TCP_FASTOPEN_CONNECT =
-            valueOf(EpollChannelOption.class, "TCP_FASTOPEN_CONNECT");
+    public static final ChannelOption<Boolean> TCP_FASTOPEN_CONNECT = ChannelOption.TCP_FASTOPEN_CONNECT;
```

Comment:
Consider add `@Deprecated` and point users to `ChannelOption.TCP_FASTOPEN_CONNECT`